CHANGELOG.md

```diff
@@ -44,6 +44,15 @@ Markdown Spec](https://github.github.com/gfm/).
 
 ### Contributors
 
+## [Unreleased] - Enabled apache mod_ssl
+[Unreleased]: https://github.com/open-oni/open-oni/compare/v#.#.#...dev
+
+### Added
+ - Enabled apache mod_ssl in web image build
+
+### Contributors
+ - Jim Campbell (lauterman)
+
```

Comment:
This looks good, but you duplicated the [Unreleased] section which will be labeled and get a description later upon release by the core maintainers. Could you restore the `v1.0.1...` part in the URL and move your Added and Contributors bullets under the existing [Unreleased] section?